### PR TITLE
Push master ca-certs to kube-system ConfigMap

### PR DESCRIPTION
**Pull request by tallclair**
**What this PR does / why we need it**:

On cluster startup, write the master's root ca-certificates to a ConfigMap (`system-cacerts`) in the `kube-system` namespace.

The purpose is to provide ca-certificates to containers that can otherwise be built on scratch. This will let us migrate several addon containers away from alpine & debian base images.

**Which issue(s) this PR fixes**:
Partially addresses https://github.com/kubernetes/kubernetes/issues/63726

Specifically, it only provides a solution for kube-system addons. This is not a general solution, and should be replaced with the general solution once one exists.

**Special notes for your reviewer**:

Are the cluster/addons manifests consumed by kubeadm or other cluster startup scripts? If so, how should we handle that?
/cc @kubernetes/sig-cluster-lifecycle-pr-reviews 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -2284,6 +2284,60 @@ EOF
...
+EOF
+}
+
+# Returns the host's ca-certificates bundle.
+# Implementation based off go's loadSystemRoots function.
+# https://golang.org/src/crypto/x509/root_unix.go
+function locate-system-cacerts {
+  # Copied from https://golang.org/src/crypto/x509/root_linux.go
+  local cacert_locations=(
+  	"/etc/ssl/certs/ca-certificates.crt"                # Debian/Ubuntu/Gentoo etc.
```
Comment (mikedanese) [inline cluster/gce/gci/configure-helper.sh]: Is this where COS stores certs?

**Comment by tallclair** [comment]
Fixed error caused by typo in method name & verified on a local cluster.
@mikedanese PTAL

**Comment by luxas** [comment]
> it only provides a solution for kube-system addons

... using kube-up in GKE on COS. Is that correct? Unless we have a generic solution we can't migrate I think.

**Comment by tallclair** [comment]
Yeah, that makes sense. It looks like we're going to end up [settling on a base image](https://github.com/kubernetes/kubernetes/issues/70249#issuecomment-440787501) that includes ca-certs, so I'm just going to abandon this for now. I still think we should consider https://github.com/kubernetes/kubernetes/issues/63726 as a long-term feature.

**Pull request closed without merging**